src/app/map/components/GoogleMaps/index.tsx

```diff
@@ -40,6 +68,84 @@ export const GoogleMaps = ({ setMapLoaded }: GoogleMapsProps) => {
               map: newMap,
             });
 
+            dummyData.forEach((event) => {
+              const eventMarker = new google.maps.Marker({
+                position:
+                  event.venue?.lat && event.venue?.lng
+                    ? { lat: event.venue.lat, lng: event.venue.lng }
+                    : null,
+                map: newMap,
+                // to add the marker
+                icon: {
+                  url: "/images/marker.svg",
+                  scaledSize: new google.maps.Size(60, 60), 
+                },
+              });
+
+              const infoWindow = new google.maps.InfoWindow();
+              {
+                event.venue?.lat &&
+                  event.venue?.lng &&
+                  infoWindow.setPosition({
+                    lat: event.venue?.lat,
+                    lng: event.venue?.lng,
+                  });
+              }
+              infoWindow.setOptions({
+                pixelOffset: new google.maps.Size(0, -50),
+              });
+
+              eventMarker.addListener("click", () => {
+                setSelectedEvent(event);
+                const eventCardHtml = ReactDOMServer.renderToString(
+                  <div className={classes.card}>
+                    <Flex direction="column" align="center" p={30}>
+                      <Image
+                        src={
+                          event?.imageUrl && event?.imageId
+                            ? `${event.imageUrl}${event.imageId}/676x380.webp`
+                            : undefined
+                        }
+                        alt={event?.title}
+                        width="100%"
+                        height={120}
+                      />
+                      <Container fz="xs" p={0}>
+                        <Text fw="bold" color={theme.colors.red[0]} mt={15}>
+                          {event?.dateTime}
+                        </Text>
+                        <Text fz="lg" fw="bold" mb={2}>
+                          {event?.title}
+                        </Text>
+                        <Text color="#999" lh={1}>
+                          <Flex align="center">
+                            <IconMapPin size="1rem" stroke={1.5} />
+                            {event?.venue?.name}
+                          </Flex>
+                        </Text>
+                        <Text mt={10} lineClamp={3}>
+                          {event?.description}
+                        </Text>
+                      </Container>
+                      {event?.eventUrl && (
+                        <Link target="_blank" href={event.eventUrl}>
+                          <Button
+                            mt={15}
+                            buttonType={BUTTON_VARIANTS.SECONDARY}
+                          >
+                            View Details
+                          </Button>
+                        </Link>
+                      )}
+                    </Flex>
+                  </div>
```

Comment:
I think we can use `EventCard` component here.
```suggestion
                  <EventCard event={event} />
```